Dockerfile

```diff
@@ -83,8 +83,8 @@ RUN \
   git config --global user.email "builder@localhost"
 
 ARG UPSTREAM_SOURCE_FALLBACK
-ENV BRVER="2022.11.1"
-ENV KVER="5.10.162"
+ENV BRVER="2023.08.4"
+ENV KVER="5.10.202"
```

Comment:
This will also bring in glibc 2.38 and add a patch to aws-lc to fix compilation errors in two tests (the patch backports changes from the tests in main-branch aws-lc).